internal/service/environment.go

```diff
@@ -23,3 +25,33 @@ func (e *Environment) ListEnvironments(ctx *context.Context, request *environmen
 
 	return response, nil
 }
+
+// CreateEnvironment creates environment
+func (e *Environment) CreateEnvironment(ctx *context.Context, request *environment.CreateEnvironmentRequest) error {
+	conn, requestCtx, err := grpcClient(ctx)
+	if err != nil {
+		return err
+	}
+	client := environment.NewEnvironmentServiceClient(conn)
+	stream, err := client.CreateEnvironment(*requestCtx, request)
+	if err != nil {
+		return err
+	}
+
+	log.Info("Creating environment...")
+	for {
+		response, err := stream.Recv()
+		if err != nil {
+			// Check for the end of the stream
+			if err == context.Canceled {
+				break
+			}
+
+			log.Fatalf("Error receiving stream: %v", err)
+		}
+
+		fmt.Println(response)
```

Comment:
1. why not use log?
2. should the response go directly to cli?

Reply:
spinner added